### DISCUSSION TITLE
Networking Issues -> in regards to ubuntu images?

### DISCUSSION THREAD

**Discussion by cloudrack-ca** [Performave/convoy#79] [Support]
Hello, Firstly I'd just like to ask if anyone else is or has been having issues with the pre-built VMTemplates That are installable from the convoypanel website. The website referred to is as followed https://convoypanel.com/docs/misc/download-vm-templates.html

The one listed with a X is the only one causing me an issue
❌ubuntu 18.04 -> 20.04 -> 22.04 VM Templates
✅windows 2019 - 2022 VM Templates
✅Debian 11 (Never failed me so far) VM Template
✅CentOS 7-8 VM Templates
✅RockyLinux 8 VM Template

Issue is when i create a ubuntu VM with said templates the networking doesnt connect im not sure why as every other VM Template Offered Works.

* Please note i am using convoy for personal use only.

**Comment by ericwang401** [comment]
Are all the Ubuntu line of images faulty?

**Reply by cloudrack-ca** [reply]
your of course speaking of the templates from https://convoypanel.com/docs/misc/download-vm-templates.html then yes

but if not please inform me as to what you mean if I'm mistaken.

**Reply by ericwang401** [reply]
Let me see if I understand. So 18.04, 20.04, and 22.04 all don't work?

**Reply by cloudrack-ca** [reply]
from what ive tested over the span of 60+ hours over 6 days yes from what I'm seeing.

**Reply by cloudrack-ca** [reply]
using virtfusion at the moment no issue but id prefer to use convoypanel i.e proxmox.

networking doesn't take

**Reply by cloudrack-ca** [reply]
@ericwang401 found the issue if you can help figure out a means to re-do the images for ubuntu 

I'm using oneprovider dedicated servers i.e they use scaleway . i.e online.net

this server must use mac address offered or generated from the panel of oneprovider for the IPv4 to work and for the VM running any ubuntu image to ping out hence my issue at hand, it seems its not setting things right in the cloudinit

- had to run a manual creation of a VM to see what was going on and why.

CT images maybe something to look into for future updates as these offer more customization i.e allowing for mac address addressing during the creation process I'm assuming the API is there but as when you create a VM and go threw the setup pop up panel (from proxmox) it doesn't provide a means for mac address just ipv4 / ipv6 + gateways..... (I did notice that convoy offers mac configuration for ip's in the panel but this seems still not to configure the mac via proxmox maybe a overlooked matter/bug)

**Reply by ericwang401** [reply]
If you could open a bug report and document all this info, I would appreciate it.

**Reply by cloudrack-ca** [reply]
Trying my best currently just performed a fresh install of PVE and convoy separated installs of course, (not convoy on PVE) with all said and done I'm not really understanding the matter 100% myself and I'm trying a few things but debian works centos works from what I'm seeing and ubuntu is not rocky works as well I might add. again something todo with cloud-init (leaning on)

**Reply by cloudrack-ca** [reply]
Internal configuration error self resolved. @ericwang401

**Reply by ericwang401** [reply]
Oh, what happened to be the issue? So I could understand the context.

**Reply by cloudrack-ca** [reply]
@ericwang401 please look at bug #83 - over the span of last night I'm not entirely sure its my pve config that's the issue hence bringing this bug up today.
